packages/webpack/src/plugins/vue/client.js

```diff
@@ -55,48 +55,45 @@ export default class VueSSRClientPlugin {
       const fileToIndex = file => manifest.all.indexOf(file)
       stats.modules.forEach((m) => {
-        // Ignore modules duplicated in multiple chunks
-        if (m.chunks.length === 1) {
-          const [cid] = m.chunks
-          const chunk = stats.chunks.find(c => c.id === cid)
-          if (!chunk || !chunk.files) {
-            return
-          }
-          const id = m.identifier.replace(/\s\w+$/, '') // remove appended hash
-          const filesSet = new Set(chunk.files.map(fileToIndex))
-
-          for (const chunkName of chunk.names) {
-            if (!entrypoints[chunkName]) {
-              const chunkGroup = namedChunkGroups[chunkName]
-              if (chunkGroup) {
-                for (const asset of chunkGroup.assets) {
-                  filesSet.add(fileToIndex(asset))
-                }
+        const [cid] = m.chunks
+        const chunk = stats.chunks.find(c => c.id === cid)
+        if (!chunk || !chunk.files) {
+          return
+        }
+        const id = m.identifier.replace(/\s\w+$/, '') // remove appended hash
+        const filesSet = new Set(chunk.files.map(fileToIndex))
+
+        for (const chunkName of chunk.names) {
+          if (!entrypoints[chunkName]) {
+            const chunkGroup = namedChunkGroups[chunkName]
+            if (chunkGroup) {
+              for (const asset of chunkGroup.assets) {
+                filesSet.add(fileToIndex(asset))
               }
             }
           }
+        }
 
-          const files = Array.from(filesSet)
-          manifest.modules[hash(id)] = files
+        const files = Array.from(filesSet)
+        manifest.modules[hash(id)] = files
 
-          // In production mode, modules may be concatenated by scope hoisting
-          // Include ConcatenatedModule for not losing module-component mapping
-          if (Array.isArray(m.modules)) {
-            for (const concatenatedModule of m.modules) {
-              const id = hash(concatenatedModule.identifier.replace(/\s\w+$/, ''))
-              if (!manifest.modules[id]) {
-                manifest.modules[id] = files
-              }
+        // In production mode, modules may be concatenated by scope hoisting
+        // Include ConcatenatedModule for not losing module-component mapping
+        if (Array.isArray(m.modules)) {
+          for (const concatenatedModule of m.modules) {
+            const id = hash(concatenatedModule.identifier.replace(/\s\w+$/, ''))
+            if (!manifest.modules[id]) {
+              manifest.modules[id] = files
             }
           }
-
-          // Find all asset modules associated with the same chunk
-          assetModules.forEach((m) => {
-            if (m.chunks.some(id => id === cid)) {
-              files.push.apply(files, m.assets.map(fileToIndex))
-            }
-          })
         }
+
+        // Find all asset modules associated with the same chunk
+        assetModules.forEach((m) => {
```

Comment:
@clarkdo Any reason not to use `for..of` here, to keep consistency with the rest of the code?